packages/unmock-core/src/service/constants.ts

```diff
@@ -1,5 +1,6 @@
 import XRegExp from "xregexp";
 
+export const DEFAULT_HTTP_METHOD = "any";
```

Comment:
I think this is a bit confusing: why is the "default HTTP method" of "service" equal to "any"? Isn't that part of the `dsl` and not inherently of service? Maybe we could consider isolating the state and DSL management to a directory of their own at some point...